source/aggregation-pipeline-builder.txt

```diff
@@ -31,6 +31,13 @@ aggregation pipeline. The :guilabel:`Preview of Documents in the
 Collection` section shows 20 documents randomly sampled from the chosen
 collection.
 
+.. note:: 
```

Comment:
Consider placing this note in the "Create an Aggregation Pipeline" section, possibly at the end of step 1 where we instruct users to choose a stage.